server/market/market_test.go

```diff
@@ -123,9 +129,16 @@ func (ta *TArchivist) failOnEpochOrder(ord order.Order) {
 	ta.mtx.Unlock()
 }
 func (ta *TArchivist) InsertEpoch(ed *db.EpochResults) error { return nil }
-func (ta *TArchivist) BookOrder(*order.LimitOrder) error     { return nil }
-func (ta *TArchivist) ExecuteOrder(ord order.Order) error    { return nil }
-func (ta *TArchivist) CancelOrder(*order.LimitOrder) error   { return nil }
+func (ta *TArchivist) BookOrder(lo *order.LimitOrder) error {
+	ta.mtx.Lock()
+	defer ta.mtx.Unlock()
+	// Note that the other storage functions like ExecuteOrder and CancelOrder
+	// do not change this order slice.
+	ta.bookedOrders = append(ta.bookedOrders, lo)
+	return nil
+}
```

Comment:
It's a touch misleading that BookOrder does this since the other methods don't have similar functionality, but this is a test...